### DISCUSSION TITLE
Besprechung der Ergebnisse

### DISCUSSION THREAD

**Discussion by Aigul99** [adrianxagigi/WissenschaftlichesArbeiten#5] [General]
Hier werden wir die Ergebnis aus Aufgabe 4, also die Analysis und Visualisierungen besprechen.

**Comment by Aigul99** [comment]
Zunächst einmal kam der Code nicht durch, weil es an geeigneten Libraries mangelte. Zweitens verstehe ich nicht ganz die Codezeile mit dem T-Test und wie man die Ergebnisse des T-Tests nutzen kann, um die Beziehungen zwischen den verschiedenen Variablen zu verstehen.

**Reply by EmonaBakalova** [reply]
Danke! Ich habe den Libraries hinzugefügt. 
Der T-Test in diesem Fall wollte ich verwenden, um festzustellen, ob es einen statistisch signifikanten Unterschied in den Ticketpreisen zwischen den verschiedenen Klassen gibt.  Es könnte auch effektiver sein, die Beziehung zwischen den Ticketpreisen und den Klassen mithilfe eines Boxplots oder andere Visualisierungsmethode darzustellen.

**Reply by Aigul99** [reply]
Aok super, ich habe eine Änderung in der ersten Visualisierung der Überlebensrate im Vergleich zu den Klassen vorgenommen. Die Methode der kategorialen Bivariate sollte nämlich verwendet werden, damit man wirklich sieht, dass es einen großen Unterschied in der Überlebensrate der ersten Klasse im Vergleich zur dritten Klasse gibt.

**Reply by Aigul99** [reply]
Ich frage mich, ob es auch einen Zusammenhang zwischen der Position des Passagiers auf der Backbord- oder Steuerbordseite und der Frage, ob er überlebt hat oder nicht, gibt.

**Reply by EmonaBakalova** [reply]
Bei vielen Passagieren fehlen die Informationen über ihre Kabinennummer. Daher ist es schwierig, genaue Schlussfolgerungen darüber zu ziehen, wo mehr Überlebende zu finden sind. Allerdings ist zu erkennen, es mehr Überlebende auf Steuerbord als auf Backbord gab.

**Comment by Aigul99** [comment]
Was haben Sie in Bezug auf die Tickets bemerkt?

**Comment by adrianxagigi** [comment]
In der zweiten Funktion: Ticketpreis nach Klasse, sieht man zwar dass durch den Boxplot, die Ticketpreise in der ersten Klasse teuerer als die in der zweiter Klasse sind. (Mit dem Median erkennt man das, der Median in der ersten Klasse ist höher als die in der zweiten Klasse) In der esten Klasse gibt es  auch viel  mehr Ausreiser, nach oben.

**Reply by Aigul99** [reply]
Ja, in der Tat, es fällt auch auf, dass es einen Unterschied zwischen den Preisen der 2. und 3. Klasse gibt, aber dieser Unterschied ist im Durchschnitt minimal.

**Comment by Aigul99** [comment]
Die Analyse der Grafiken zeigt, dass es mehr weibliche Überlebende gab als männliche.

**Reply by adrianxagigi** [reply]
In der ersten und der zweiten Klasse haben fast alle Frauen überlebt. In der dritten Klasse sind fast alle Männer gestorben, und mehr von der ersten Klasse haben überlebt als die, die gestorben sind.

**Comment by EmonaBakalova** [comment]
Ticketpreis und Überlebende: Wir haben den t-Test benutzt. Der P-Wert ist sehr klein (2.699e-11) und der Unterschied in den Mittelwerten ist statistisch signifikant. Der t-Test legt nahe, dass es einen signifikanten Unterschied im durchschnittlichen Fahrpreis zwischen Passagieren gibt, die überlebt haben, und denen, die es nicht haben. Im Durchschnitt ist der Fahrpreis für Überlebende signifikant höher als für Nicht-Überlebende.
